src/Commands/CopyTemplatesCommand.php

```diff
@@ -52,46 +52,84 @@ public function copyTemplates(string $site_name) {
-            $contents = str_replace(
-              '**HASH_SALT**',
-              Crypt::randomBytesBase64(55),
-              $contents
-            );
-            file_put_contents($clone_dir . '/' . $iterator->current()
-                ->getFilename(), $contents);
-            break;
-
-          default:
-            copy(
-              $iterator->current()->getRealPath(),
-              $clone_dir . '/' . $iterator->current()->getFilename()
-            );
-        }
-      }
-    }
+
+    $this->copyFrameworkFiles( $this->getFramework( $site_name ), $site_name, $base_dir, $clone_dir );
     $this->processGitIgnore($clone_dir);
+
     exec('direnv allow');
     if (php_uname("s") == "Darwin") {
       exec('brew bundle install');
     }
     echo (file_get_contents($base_dir . "/docs/demigod.txt"));
   }
 
+  /**
+   * Copy the template files based on the framework.
+   *
+   * @param string $framework The CMS framework identified by getFramework().
+   * @param array ...$args Array of arguments required for copying files.
+   *              $site_name The site name that was called into copyTemplates.
+   *              $base_dir This plugin's base directory.
+   *              $clone_dir The directory the site was cloned into.
+   */
+  private function copyFrameworkFiles( string $framework, ...$args ) {
+    [ $site_name, $base_dir, $clone_dir ] = $args;
+    $iterator = new \DirectoryIterator("$base_dir/templates/$framework");
+    for ($iterator->rewind(); $iterator->valid(); $iterator->next()) {
+        if (is_file($iterator->current()->getRealPath())) {
+          switch ($iterator->current()->getFilename()) {
+            case 'settings.local.php':
+              copy(
+                  $iterator->current()->getRealPath(),
+                  $clone_dir . '/web/sites/default/settings.local.php'
+              );
+              break;
+
+            case '.envrc':
+              $contents = file_get_contents($iterator->current()->getRealPath());
+              $contents = str_replace('**PROJECT_NAME**', $site_name, $contents);
+              $contents = str_replace('**PROJECT_PATH**', $clone_dir, $contents);
+              $contents = str_replace(
+                '**HASH_SALT**',
+                Crypt::randomBytesBase64(55),
+                $contents
+              );
+              file_put_contents($clone_dir . '/' . $iterator->current()->getFilename(), $contents);
+              break;
+
+            default:
+              copy(
+                  $iterator->current()->getRealPath(),
+                  $clone_dir . '/' . $iterator->current()->getFilename()
+              );
+          }
+        }
+    }
+  }
+
+  /**
+   * Get the site framework.
+   *
+   * @command demigod:get-framework
+   *
+   * @param string $site_name The site name to query.
+   *
+   * @return string The site framework, pulled from site:info.
+   */
+  public function getFramework( string $site_name ) : string {
```

Comment:
If the framework is any Drupal, the copy files function fails because the version number is part of the framework. This needs to return simply either `wordpress` if WordPress or `drupal` if anything else.